src/components/CreateCollectionView.js

```diff
@@ -2,13 +2,13 @@ import React from "react";
 import List from "./List";
 import Alerts from "./Alerts";
 import * as db from "../util/dbfunctions";
-import PlanFunctionBar from "./PlanFunctionBar";
+import CreateFunctionBar from "./CreateFunctionBar";
 
-class PlanSessionView extends React.Component {
+class CreateCollectionView extends React.Component {
```

Comment:
Here you have refactored to call them collections but I believe you still call them sessions and templates in the app. Why the name discrepancy?

Reply:
Collection was the name I was using on the backend to reference both Sessions and Templates, as they are both a type of collection I guess. The function of that page now works for both Sessions and Templates, so I felt the previous name was now not applicable